src/create_disk.sh

```diff
@@ -339,7 +338,21 @@ install_uefi() {
             done
         )
     done
+}
 
+install_uefi() {
+    if test -d ${deploy_root}/usr/lib/bootupd/updates; then
+        # https://github.com/coreos/fedora-coreos-tracker/issues/510
+        # See also https://github.com/ostreedev/ostree/pull/1873#issuecomment-524439883
+        /usr/libexec/bootupd install --src-root=${deploy_root} ${rootfs}
```

Comment:
nit: the use of `${deploy_root}` and `${rootfs}` should be double-quoted

```suggestion
    if test -d "${deploy_root}/usr/lib/bootupd/updates"; then
        # https://github.com/coreos/fedora-coreos-tracker/issues/510
        # See also https://github.com/ostreedev/ostree/pull/1873#issuecomment-524439883
        /usr/libexec/bootupd install --src-root="${deploy_root}" "${rootfs}"
```

Reply:
Thanks, fixed.